### DISCUSSION TITLE
Display Bandwidth usage

### DISCUSSION THREAD

**Discussion by dmitrym0** [FelixKratz/SketchyBar#253] [Q&A] [answered]
Hey folks,

Just curious if anyone has any idea on how to show bandwidth in the bar? Something like https://github.com/exelban/stats?

Are there any console tools that can be leverage for this?

**Comment by FelixKratz** [comment, marked as answer]
There are two approaches:

* A script like this: https://github.com/FelixKratz/SketchyBar/discussions/12?sort=old#discussioncomment-1377134 using `ifstat`
* Displaying the Stats menu item itself in sketchybar: https://felixkratz.github.io/SketchyBar/config/components#item-alias----mirror-items-of-the-original-macos-status-bar-into-sketchybar

**Reply by dmitrym0** [reply]
Thanks so much. I missed item aliasing in the docs.
